internal/commands/secrets/output.go

```diff
@@ -1,6 +1,56 @@
 package secrets
 
+import (
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
 const (
-	headerID   = "ID"
-	headerName = "Name"
+	headerID      = "ID"
+	headerName    = "Name"
+	headerDeleted = "Deleted"
+	headerDetails = "Details"
+
+	secretDeleteMessage = "Deleted Secrets"
 )
+
+type secretOutputs []secretOutput
+
+type secretOutput struct {
+	secret realm.Secret
+	err error
+}
+
+type secretTableRowModifier func(secretOutput, map[string]interface{})
+
+func secretOutputComparerBySuccess(outputs secretOutputs) func(i, j int) bool{
+	return func(i, j int) bool {
+		return outputs[i].err != nil && outputs[j].err == nil
+	}
+}
+
+func secretHeaders() []string{
```

Comment:
check out the latest code in the `user` package with how they're handling `headers` and `rows`.  I think it super close to the way you have it here, but can we adjust this approach so it can be similarly used in `secrets list`?